src/Utils/messages-media.ts

```diff
@@ -4,6 +4,7 @@ import { exec } from 'child_process'
 import * as Crypto from 'crypto'
 import { once } from 'events'
 import { createReadStream, createWriteStream, promises as fs, WriteStream } from 'fs'
+import { loadEsm } from 'load-esm'
```

Comment:
its possible to not use `loadEsm`?

Reply:
Seems to be a way to tweak TypeScript to avoid this conversion issue, and to it without [loadEsm()](https://github.com/Borewit/load-esm), see: #1355